### PR TITLE
DL notebook fixes

### PR DESCRIPTION
**Pull request by Imraj-Singh**
Dealing with a couple errors when running without GPU. One being `print(torch.cuda.get_device_name(0))` is called even when installed without cuda, the other `checkpoint = torch.load('trained.torch_model')` mapped to GPU.

Also removing requires_grad_() as per Georg's comment.

### CODE REVIEW

**Comment by KrisThielemans** [comment]
Just ping me or @evgueni-ovtchinnikov when you want this to be merged.

**Comment by KrisThielemans** [comment]
best to add a line to CHANGES.md

**Comment by Imraj-Singh** [comment]
@KrisThielemans added a line to changes.md think it's fine to merge. They're very minor changes

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 ## v3.6.0
 * moved gradient_descent_mr_pet_ct.ipynb to Introductory folder and fixed some issues such that it does gradient descent for all modalities.
 * updated devcontainer to pull docker images from ghcr.io
+* small updates to DL notebooks
```
Comment (KrisThielemans) [inline CHANGES.md:5]:
```suggestion
* small updates to DL notebooks [#222](https://github.com/SyneRBI/SIRF-Exercises/pull/222)
```

Comment (KrisThielemans) [reply CHANGES.md:5]: otherwise not too informative :-/

**Review by KrisThielemans — CHANGES_REQUESTED** [review]

**Pull request merged**